src/main/java/com/github/tomakehurst/wiremock/matching/EqualToUrlEncodedFormPattern.java

```diff
@@ -22,7 +23,7 @@
 
     public EqualToUrlEncodedFormPattern(@JsonProperty("equalToUrlEncodedForm") String urlEncodedForm) {
         super(urlEncodedForm);
-        this.charset = null;
+        this.charset = StandardCharsets.ISO_8859_1.name();
```

Comment:
this charset is the default if headers don't contain "charset" param & works empirically - please let me know if something different is needed for compatibility with different server versions

Reply:
Looks good to me!